Makefile

```diff
@@ -47,3 +48,6 @@ push: ./manifest-tool $(addprefix push-,$(ALL_ARCH))
 ./manifest-tool:
 	curl -sSL https://github.com/estesp/manifest-tool/releases/download/v0.5.0/manifest-tool-linux-amd64 > manifest-tool
 	chmod +x manifest-tool
+
+vendor: glide.lock
+	glide install -v
```

Comment:
heh, I do the opposite -- I don't have Docker on my host system (it's in a bunch of VMs), so I tend to get a bit peeved when makefiles try to run docker.  I'll try to figure something out for this.